lib/src/store.dart

```diff
@@ -58,22 +63,36 @@ class Store {
   /// As an example, [transformer] could be used to throttle the number of
   /// triggers this [Store] emits for state that may update extremely frequently
   /// (like scroll position).
-  Store.withTransformer(StreamTransformer<dynamic, dynamic> transformer) {
-    _streamController = new StreamController<Store>();
+  Store.withTransformer(StreamTransformer<dynamic, dynamic> transformer)
+      : _streamController = new StreamController<Store>() {
+    manageStreamController(_streamController);
 
     // apply a transform to the stream if supplied
     _stream = _streamController.stream
         .transform(transformer as StreamTransformer<Store, dynamic>)
```

Comment:
That's a good point. I could see the argument that something like scroll position should be throttled earlier since otherwise it's just an unnecessary volume of stream events, even if they don't make it all the way to forcing a redraw, but either way, it's probably something that we can safely leave to consumers